ProjectTodo/src/services/CardsService.js

```diff
@@ -0,0 +1,27 @@
+import BaseService from './BaseService';
+import { AsyncStorage } from 'react-native';
+
+const ENDPOINTS = {
+  GET_CARDS: '/cards',
+};
+
+class CardsService extends BaseService {
+
+  constructor(props){
+    super(props);
+  }
+
+  getCards = async token => {
+    try {
+        var response = this.apiClient().get(ENDPOINTS.GET_CARDS);
+        response.then( resolve => console.log(resolve.data))
+        .catch( error => console.log("GRESKA" + error));
```

Comment:
console.log user ne vidi :) to je vise za tebe za debagovanje tako da ga ga skloni gde god ga imas